build/templates/_converters.py.mako

```diff
@@ -0,0 +1,32 @@
+<%
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+%>\
+from ${module_name} import visatype
+
+
+def _timedelta_converter(value, library_type, scaling):
```

Comment:
These are functions, so their names should be verbs, not nouns.

* _convert_timedelta
* convert_timedelta_to_seconds
* convert_timedelta_to_milliseconds
* convert_timedelta_to_microseconds

Reply:
Fixed